llvm/test/Verifier/global-ctors-dtors-uses.ll

```diff
@@ -1,16 +0,0 @@
-; RUN: not llvm-as %s -o /dev/null 2>&1 | FileCheck %s
```

Comment:
would be better to turn this into a positive test instead of deleting it, no?

Reply:
Thanks for suggestion. I've added llvm/test/CodeGen/Generic/global-ctors-dtors-uses.ll since Verifier subdir does not look as a right place for such a positive test.